mod_reforged/hooks/skills/actives/lesser_flesh_golem_attack_skill.nut

```diff
@@ -0,0 +1,14 @@
+::Reforged.HooksMod.hook("scripts/skills/actives/lesser_flesh_golem_attack_skill", function(q) {
+	q.create = @(__original) function()
+	{
+		__original();
+		// Vanilla is missing a description for this skill
+		this.m.Description = ::Reforged.Mod.Tooltips.parseString("A blunt melee attack that dazes the target on hit if they are not immune to stun.");
```

Comment:
`[dazes|Skill+dazed_effect]` and `not immune to being [stunned|Skill+stunned_effect]`